README.adoc

```diff
@@ -21,17 +21,27 @@ You'll explore how to provide system and application metrics from a microservice
 
 You will learn how to use MicroProfile Metrics to provide metrics from a RESTful application.
 Include metrics in an application to pinpoint issues, provide long-term trend data for
-capacity planning, and provide proactive discovery of issues, such as disk usage growth without bounds. Metrics can
-also help you schedule systems and decide when to scale the application to run on more or fewer machines.
+capacity planning, and provide proactive discovery of issues, such as disk usage growth without
+bounds. Metrics can also help you schedule systems and decide when to scale the application to
+run on more or fewer machines.
 
-Use the MicroProfile Metrics endpoint to get metrics
-results in a plain text format. The endpoint of an application scope includes a type field with counter, gauge, and timer types.
+The application that you will work with is an `inventory` service
+that stores information about various JVMs that run on different systems.
+Whenever a request is made to the `inventory` service to retrieve the JVM
+system properties of a particular host, the `inventory` service communicates with the `system`
+service on that host to get the system properties. The system properties are then stored and returned.
 
-You will work with an `inventory` service, which stores information
-about various JVMs that run on different systems.
+Use MicroProfile Metrics to provide endpoints where you can get the metrics
+results in a plain text format or JSON format. These endpoints have three scopes: `base`,
+`application`, and `vendor`. The `application` scope is the focus for `inventory` application.
 
-You will apply the 'metrics type annotations' to the inventory application to provide metrics
-at the application scope.
+The metadata of the metrics in any scope has fields, such as unit, type, description, and more.
+The type can be a counter, gauge, meter, histogram, or timer. You will use the counter, gauge, and
+timer types in your application.
+
+You can instrument an application in different ways, such as with injection, annotations, or by using
+the `MetricRegistry` directly. You will learn how to use annotations to register metrics to the
```

Comment:
For the second sentence, I think it should be scoped to the guide.

_In this guide, you will learn how to use annotations to register metrics to the..._

Reply:
@sishida Can we use "in this guide" or something similar ?

Reply:
I can see how we'd want it here since we're explaining general info and then going back to the specifics of the guide. So, yes, "In this guide, you will learn..." is OK.

Reply:
Done